### DISCUSSION TITLE
【求助】想问一下大伙

### DISCUSSION THREAD

**Discussion by dolokong** [share121/inter-knot#2450] [其他]
想问一下大伙，刚才看见其他帖子有人评论区里发表情包，但是我尝试了好几次发图片结果只能出来网址，这是怎么回事？如何才能正确发送表情包？我尝试提取了其他人表情包的链接，发现很多都是米游社的。

**Comment by Qweulin** [comment]
这个[文章](https://share121.github.io/inter-knot/?article=2049)，记得给作者仓库一个star

**Comment by ysd1911** [comment]
绳网里面有个用米游社表情包的脚本，你把那个脚本下上就能用了

**Comment by isle200inf** [comment]
这个委托我接了！
